package.json

```diff
@@ -113,7 +113,6 @@
     "govuk-frontend": "^5.9.0",
     "helmet": "^8.1.0",
     "http-errors": "^2.0.0",
-    "jquery": "^3.7.1",
```

Comment:
We only had `jQuery` because MoJ front-end previously required it - so removing and tidying references in other files.